tests/scripts/thread-cert/Cert_5_1_03_RouterAddressReallocation.py

```diff
@@ -32,26 +32,49 @@
 import mle
 import network_layer
 import thread_cert
+from pktverify.consts import MLE_ADVERTISEMENT, MLE_PARENT_REQUEST, MLE_CHILD_ID_REQUEST, ADDR_SOL_URI, MODE_TLV, TIMEOUT_TLV, CHALLENGE_TLV, RESPONSE_TLV, LINK_LAYER_FRAME_COUNTER_TLV, ROUTE64_TLV, ADDRESS16_TLV, LEADER_DATA_TLV, NETWORK_DATA_TLV, TLV_REQUEST_TLV, SCAN_MASK_TLV, VERSION_TLV, ADDRESS_REGISTRATION_TLV, NL_MAC_EXTENDED_ADDRESS_TLV, NL_STATUS_TLV
+from pktverify.packet_verifier import PacketVerifier
 
 LEADER = 1
 ROUTER1 = 2
 ROUTER2 = 3
 
+# Test Purpose and Description:
+# -----------------------------
+# The purpose of this test case is to verify that after the removal of the
```

Comment:
I am thinking the purpose should be appended as `then attach to a new partition, attempting to allocate its original short address` might be more appropriate to better reflect the case name `RouterAddressAllocation`.

Reply:
added DUT request its original short address into test purpose and checks